### PR TITLE
Extend the login session timeout to 2 days

### LINKED ISSUE #329: Extend the time limit of the logged in user session
**Is your feature request related to a problem? Please describe.**
The users get logged out of platform automatically after a certain time period because of the session expiration. Need to extend the time limit to a comfortable number. 

**Definition of done**
Users are able to login and stay logged in for 2 - 3 days without having to log in through Linkedin again

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #329 

## Goals

Users being able to log in and stay logged in for 2 days without having to log in through Linkedin again

## Approach

Increased the session timeout to 2 days
Increased the session cookie max-age to 2 days

### Screenshots

N/A

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
macOS Monterey
AdoptOpenJDK 8

Manually tested by setting 1 day as the timeout

## Learning

N/A

### CODE REVIEW

File: src/main/resources/application.yml.example [hunk 1]
```diff
@@ -43,3 +43,9 @@ spring:
             enable: true
   jmx:
     unique-names: true
+server:
+  servlet:
+    session:
+      cookie:
+        max-age: 1d
+      timeout: 1d
```
Comment (Gravewalker666) [inline src/main/resources/application.yml.example:51]: did you mean `2d`?

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml.example (hunk 1 above)
Comment (Piumal1999) [reply src/main/resources/application.yml.example:51]: oh yes. I'll update it

Comment (Piumal1999) [reply src/main/resources/application.yml.example]: Updated with 1830da4

**Review by Gravewalker666 — APPROVED** [review]
awesome

**Pull request merged**